xapian-applications/omega/index_file.cc

```diff
@@ -172,6 +172,11 @@ index_add_default_libraries()
     index_library("application/vnd.apple.pages", omindex_libetonyek);
     index_library("application/vnd.apple.numbers", omindex_libetonyek);
 #endif
+#if defined HAVE_TESSERACT
+    Worker* omindex_tesseract = new Worker("omindex_tesseract");
+    index_library("image/jpeg", omindex_tesseract);
+    index_library("image/png", omindex_tesseract);
```

Comment:
It looks like there are other common image formats supported, though expanding this list could be handled as a follow-on change.

Also, we should have mappings from the extensions used for these formats (`jpeg`, `jpg`, and `png`) to the MIME content-types in `mimemap.tokens` so we don't have to call libmagic to find out what type they are.

Reply:
Yes, [here](https://tpgit.github.io/UnOfficialLeptDocs/leptonica/README.html#supported-image-file-formats) is the "list" of supported formats. I am going to add them to `mimemap.tokens`

Reply:
You've added all the listed entries from there except BMP - was there a reason to exclude that?

(I get `image/x-ms-bmp` from libmagic for it.)